native/cocos/editor-support/spine/AnimationState.h

```diff
@@ -230,8 +230,9 @@ class SP_API TrackEntry : public SpineObject, public HasRendererObject {
     void setListener(AnimationStateListener listener);
 
     void setListener(AnimationStateListenerObject* listener);
-
+#ifndef __EMSCRIPTEN__
 private:
+#endif
```

Comment:
Why need it?

Reply:
For wasm/asmjs binding, we bind the private/protected properties directly which could reduce package size,
since the `setter/getter` function only does `property assignment` and `returning property`.